static/app/views/replays/detail/breadcrumbs/breadcrumbItem.tsx

```diff
@@ -93,32 +104,39 @@ type CrumbItemProps = {
 
 const CrumbItem = styled(PanelItem)<CrumbItemProps>`
   display: grid;
-  grid-template-columns: max-content max-content auto max-content;
-  align-items: center;
+  grid-template-columns: max-content auto;
+  align-items: flex-start;
   gap: ${space(1)};
   width: 100%;
 
   font-size: ${p => p.theme.fontSizeMedium};
   background: transparent;
-  padding: 0;
-  padding-right: ${space(1)};
+  padding: ${space(1)};
   text-align: left;
-
   border: none;
-  border-bottom: 1px solid ${p => p.theme.innerBorder};
-  ${p => p.isHovered && `background: ${p.theme.surface400};`}
-
-  /* overrides PanelItem css */
-  &:last-child {
-    border-bottom: 1px solid ${p => p.theme.innerBorder};
-  }
+  position: relative;
+  ${p => p.isSelected && `background-color: ${p.theme.purple100};`}
+  ${p => p.isHovered && `background-color: ${p.theme.surface100};`}
+  border-radius: 4px;
 
-  /* Selected state */
-  ::before {
+  &::after {
     content: '';
-    width: 4px;
+    position: absolute;
+    left: 19.5px;
+    width: 1px;
+    background: ${p => p.theme.gray200};
     height: 100%;
-    ${p => p.isSelected && `background-color: ${p.theme.purple300};`}
+    z-index: 1;
+  }
+
+  &:first-of-type::after {
+    bottom: 0;
+    height: 50%;
+  }
+
+  &:last-of-type::after {
+    top: 0;
+    height: 50%;
```

Comment:
This could probably be something like `8px` (well, anything up to 30px)  because we just need to cover the top padding.